### DISCUSSION TITLE
Failed to resolve import "#imports"

### DISCUSSION THREAD

**Discussion by megvargian** [vitest-dev/vitest#4353] [Q&A]
Hello, 

I have a question regarding auto imports in the Nuxt 3 unit test with Vitest. I am trying to mount a Component that has this import "import { useLocalePath } from '#imports';" and somehow Vite or Vitest doesn't recognize whether the file exists ( the actual file location is in ".nuxt/imports"). I tried with libraries like unplugin-auto-import and unplugin-vue-components and configured them in vitest.config.ts, but it didn't work still giving me the same error message. Am I missing something here? Is there a way to solve this issue?

Thank you in advance 
Mike

**Comment by gyofeel** [comment]
I have a same issue.
https://github.com/vitest-dev/vitest/discussions/4741
